src/api/index.ts

```diff
@@ -17,6 +17,10 @@ router.get("/ping", (_req, res) => {
   res.json(createSuccessStringResponse("abap is forevah!"));
 });
 
+router.get("/get_default_configuration", (_,res) => {
+  res.json(getDefaultConfig());
```

Comment:
Maybe createSuccessResponse<...>(getDefaultConfig()) to keep the api shape?

Reply:
Changed it - let me know if that's how you meant it

Reply:
Yeap, this look fine. Would you mind to make a short unit test for this as well ? Example is in https://github.com/abaplint/abaplint-cloud-foundry/blob/master/test/check-object.api.test.ts

No need to check the config of course, just it's presence like in check-object test. With `toMatchObject`

Reply:
Added test case for API and moved config stuff to new file